### PR TITLE
Robots can now remove tape from cameras using their gripper

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Lets robots remove tape from cameras using their magnetic gripper
Also gives it  descriptions for antags and normal

## Why It's Good For The Game
Firefox approved

## Changelog
:cl:
balance:Robots can now remove tape from cameras using their magnetic gripper
/:cl:

### CODE REVIEW

**Comment by garrulousTeamotei** [comment]
Misspelled 'Remove' as 'remote' on the description

**Comment by MLGTASTICa** [comment]
fixed

**Comment by ArmolitskiyCarpoeb** [comment]
Bad (i hate robots)

File: code/modules/mob/living/silicon/robot/gripper.dm [hunk 1]
```diff
@@ -159,6 +161,14 @@
 	if(wrapped) //Already have an item.
 		return//This is handled in /mob/living/silicon/robot/GripperClickOn
 
+	if(istype(target, /obj/machinery/camera) && user.a_intent == I_HELP)
+		var/obj/machinery/camera/cam = target
+		if(cam.taped)
+			to_chat(user, SPAN_NOTICE("You remove the tape from the [cam] using the edge of your magnetic gripper"))
```
Comment (hyperioo) [inline code/modules/mob/living/silicon/robot/gripper.dm:167]:
```suggestion
			to_chat(user, SPAN_NOTICE("You remove the tape from \the [cam] using the edge of your magnetic gripper."))
```

File: code/modules/mob/living/silicon/robot/gripper.dm [hunk 2]
```diff
@@ -3,6 +3,8 @@
 /obj/item/gripper
 	name = "magnetic gripper"
 	desc = "A simple grasping tool specialized in construction and engineering work."
+	description_info = "Can be used to remove sticky tape from cameras on help intent"
+	description_antag = "Can be used for a strong 20 brute attack on harm intent on humans"
```
Comment (hyperioo) [inline code/modules/mob/living/silicon/robot/gripper.dm:7]:
```suggestion
	description_info = "Can be used to remove sticky tape from cameras on help intent."
	description_antag = "Can be used for a strong brute attack on humans using harm intent."
```

**Review by hyperioo — APPROVED** [review]

**Pull request merged**